server/src/core/person/person.resolver.ts

```diff
@@ -156,4 +164,31 @@ export class PersonResolver {
       select: prismaSelect.value,
     } as Prisma.PersonCreateArgs);
   }
+
+  @Mutation(() => String)
+  async uploadPersonPicture(
+    @Args('id') id: string,
```

Comment:
let's add permission check here too
  @UseGuards(AbilityGuard)
  @CheckAbilities(UpdatePersonAbilityHandler)

Reply:
And check that we cannot modify the picture of a person out of the user's workspace